src/database/index.tsx

```diff
@@ -123,6 +119,68 @@ const buildUpdatePayload = (entity: any) => {
   return payload;
 };
 
+const timestampToDate = (timestamp: firebase.firestore.Timestamp): Date => {
+  return timestamp.toDate();
+};
+
+const buildPlannedRelease = (plannedReleaseData: any): PlannedRelease => {
+  let plannedRelease: PlannedRelease = plannedReleaseData;
+
+  const plannedReleaseDataDate = plannedReleaseData.date;
+  if (plannedReleaseDataDate instanceof firebase.firestore.Timestamp) {
```

Comment:
This type check allows us to operate with the existing planned release string dates without the need for a data migration